src/dms/core/EdFi.DataManagementService.Core.External/Model/AuthorizationStrategyFilter.cs

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.DataManagementService.Core.External.Model;
+
+/// <summary>
+/// Represents collection of authorization filters and filter operator
+/// </summary>
+public record AuthorizationStrategyFilter(
```

Comment:
I'd like better naming to differentiate between an `AuthorizationFilter` and `AuthorizationStrategyFilter`.